astronomer/providers/amazon/aws/example_dags/example_aws_nuke.py

```diff
@@ -29,6 +29,15 @@
 ) as dag:
     start = DummyOperator(task_id="start")
 
+    terminate_running_emr_virtual_clusters = BashOperator(
+        task_id="terminate_running_emr_virtual_clusters",
+        bash_command=f"set -e; "
+        f"aws configure set aws_access_key_id {AWS_ACCESS_KEY_ID}; "
```

Comment:
Do we need to install aws-cli?

Reply:
If we already have it installed, I'm good with this PR. Thanks for taking care of this issue!

Reply:
yes, it's installed already with the images we build using the Dockerfiles. e.g. Dockerfile.aws_cloud in the integration_tests or dev/Dockerfile.aws